src/components/map/components/mobility/Vehicles.tsx

```diff
@@ -82,11 +82,11 @@ export const Vehicles = ({mapCameraRef, vehicles, onPress}: Props) => {
         minZoomLevel={13.5}
         belowLayerID="clusterIcon"
         style={{
-          circleColor: '#920695',
-          circleStrokeColor: '#920695',
-          circleOpacity: 0.7,
-          circleStrokeOpacity: 0.2,
-          circleStrokeWidth: ['min', ['+', 2, ['get', 'point_count']], 12],
+          circleColor: '#4A753A',
+          circleStrokeColor: '#4A753A',
```

Comment:
Should this be on the design system instead? Or the remote config?

Reply:
Might be something to take into consideration. It is hard to visualise how these colors are shown in the app just by doing the PR, but every color that a transport company might want to change should be in the design system.